decidim-proposals/lib/decidim/proposals/seeds.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "decidim/components/namer"
+require "decidim/faker/localized"
 
 module Decidim
   module Proposals
```

Comment:
This will cause a load error on faker when using prod environment. 
The above file is requesting faker.

```suggestion
require "decidim/components/namer"

module Decidim
  module Proposals
```

Reply:
That bug would be fixed with #12029, as it's something that will happen with other modules and it isn't something specific to proposals